src/pkg/packager/dev.go

```diff
@@ -105,3 +107,46 @@ func (p *Packager) DevDeploy(ctx context.Context) error {
 	// cd back
 	return os.Chdir(cwd)
 }
+
+// Lint ensures a package is valid & follows suggested conventions
+func (p *Packager) Lint(ctx context.Context) (err error) {
+	config.CommonOptions.Confirm = true
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	if err := os.Chdir(p.cfg.CreateOpts.BaseDir); err != nil {
+		return fmt.Errorf("unable to access directory %q: %w", p.cfg.CreateOpts.BaseDir, err)
+	}
+
+	pc := creator.NewPackageCreator(p.cfg.CreateOpts, cwd)
+
+	if err := helpers.CreatePathAndCopy(layout.ZarfYAML, p.layout.ZarfYAML); err != nil {
+		return err
+	}
+
+	p.cfg.Pkg, p.warnings, err = pc.LoadPackageDefinition(ctx, p.layout)
+	if err != nil {
+		return err
+	}
+
+	lintFindings, err := lint.Validate(ctx, p.cfg.CreateOpts)
+	if err != nil {
+		return fmt.Errorf("linting failed: %w", err)
+	}
+	p.warnings = append(p.warnings, lintFindings...)
+
+	if len(p.warnings) == 0 {
+		message.Successf("0 findings for %q", p.cfg.Pkg.Metadata.Name)
+		return nil
+	}
+
+	lint.PrintFindings(p.warnings, types.SevWarn, p.cfg.CreateOpts.BaseDir, p.cfg.Pkg.Metadata.Name)
+
+	if lint.HasErrors(p.warnings) {
```

Comment:
Same comments as above

Reply:
Yeah I don't love this, but I still want to give an error to cobra so that it exits with 1. Here is it with the error message vs without. I'm fine with either way. 
![image](https://github.com/defenseunicorns/zarf/assets/37223396/e0b4b399-5c67-46c4-b13b-ea84b945b2aa)